curvefs/src/client/curve_fuse_op.cpp

```diff
@@ -256,6 +258,15 @@ void QueryWarmupTask(fuse_ino_t key, std::string *data) {
 }
 
 int Warmup(fuse_ino_t key, const std::string& name, const std::string& value) {
+    /*
+     * value[0]: WarmupOpType: add, single
+     * value[1]: WarmupType: single, list
+     * value[2]: CurvefsPath
+     * value[3]: StorageType
+     * value[4]: MountPoint
+     * value[5]: Root
```

Comment:
Do CurvefsPath and root refer to the same thing?

Reply:
No.
`CurvefsPath` refers to the present file (for which we set an xattr), denoting its path within curvefs. 
On the other hand, `root` refers to the root within curvefs.

Reply:
These positions can be defined as constants

Reply:
defined